src/slicer/SlicerArrows.js

```diff
@@ -0,0 +1,187 @@
+import Cartesian3 from 'cesium/Source/Core/Cartesian3';
+import ShadowMode from 'cesium/Source/Scene/ShadowMode';
+import ColorBlendMode from 'cesium/Source/Scene/ColorBlendMode';
+import {SLICING_GEOMETRY_COLOR} from '../constants';
+import CallbackProperty from 'cesium/Source/DataSources/CallbackProperty';
+import Entity from 'cesium/Source/DataSources/Entity';
+import Color from 'cesium/Source/Core/Color';
+import BoundingSphere from 'cesium/Source/Core/BoundingSphere';
+import Cartesian2 from 'cesium/Source/Core/Cartesian2';
+import ScreenSpaceEventType from 'cesium/Source/Core/ScreenSpaceEventType';
+import JulianDate from 'cesium/Source/Core/JulianDate';
+import ScreenSpaceEventHandler from 'cesium/Source/Core/ScreenSpaceEventHandler';
+import {getDirectionFromPoints} from '../cesiumutils';
+
+/**
+ * @typedef {object} ArrowListItem
+ * @property {string} side - arrow position label
+ * @property {string} uri - path to model
+ * @property {string} [oppositeSide - opposite arrow position label]
+ * @property {string} [position - arrow position. Required if 'positionUpdateCallback' is not provided]
+ * @property {string} [oppositePosition - position to create move axis. Required if no opposite arrow]
+ */
+
+export default class SlicerArrows {
+  /**
+   * Creates one or more entities and handle their move.
+   * @param {Viewer} viewer
+   * @param {DataSource} dataSource - dataSource to store entities
+   * @param {{
+   *          arrowsList: ArrowListItem[],
+   *          positionUpdateCallback: (function(string): Cartesian3),
+   *          moveCallback: (function(string, number, Cartesian3): void)
+   *        }} options
```

Comment:
You should create a typedef like you did a few lines above, not put everything inline.